### PR TITLE
Portfolio Deployment

### PR DESCRIPTION
**Pull request by Bellagirl-maker**
I deployed my portfolio and updated the readme file with the live link of my site
Here is a link to my live site
https://bellagirl-maker.github.io/Portfolio/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -108,7 +108,7 @@ Project section was buiilt with HTML and CSS
 
 > Add a link to your deployed project.
 
-- [Live Demo Link](https://yourdeployedapplicationlink.com)
+- [Live Demo Link](https://bellagirl-maker.github.io/Portfolio/)
```
Comment (AlphaNtihinduka) [inline README.md:108]: - [ ] You have added the link in the README. However, you need to ensure that the link is visible in the README file of the PR description ☺️ .

**Review by AlphaNtihinduka — CHANGES_REQUESTED** [review]
## Hi @Bellagirl-maker 🙋‍♂️
Good job working on implementing the github deployment. You have done the design to do the expectation and incredibly well. 👏  

TO HIGHLIGHT:
- Your project is professional ✅
- Professional README file :100: 
- Your PR is not yet professional ✅
- Well-structured files ✅

# STATUS: Required Changes :recycle: 

close

## Required changes :recycle: 
- Kindly consider the inline comments.

## Optional suggestions
- [ ] [OPTIONAL]  


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or slack channel if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HAPPY CODING AND GOOD LUCK :crossed_fingers: :four_leaf_clover:

File: README.md (hunk 1 above)
Comment (Bellagirl-maker) [reply README.md:108]: @AlphaNtihinduka The link is now visible in the README file of the PR description.

**Review by wayungi — APPROVED** [review]
Hi @Bellagirl-maker , 
image

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- [ ] All linter checks passing ✔️ 
- [ ] Good commit messages ✔
- [ ] No merge conflicts ✔️ 
- [ ]  Correct GitHub flow ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**